build/Targets/Dependencies.props

```diff
@@ -15,7 +15,7 @@
     <MicrosoftBuildTasksCoreVersion>15.1.0-preview-000458-02</MicrosoftBuildTasksCoreVersion>
```

Comment:
Not part of this PR strictly speaking, but we do have an RTW package available for this dependency now.